docs/getting_started.md

```diff
@@ -131,10 +134,11 @@ An example repository containing a guestbook application is available at
 
 ### Creating Apps Via CLI
 
-!!! note
-    You can access Argo CD using port forwarding: add `--port-forward-namespace argocd` flag to every CLI command or set `ARGOCD_OPTS` environment variable: `export ARGOCD_OPTS='--port-forward-namespace argocd'`:
+Register the guestbook repository with the ArgoCD controller with the following command:
```

Comment:
I think this should be rephrased to something like:

> Create the example guestbook application with the following command:

Reply:
Thanks, updated!